configs/vitest.config.e2e.ts

```diff
@@ -19,6 +19,10 @@ export const e2eProject = defineProject({
         singleFork: true,
       },
     },
+    sequence: {
+      concurrent: false,
+      shuffle: false,
+    },
```

Comment:
This is not a behavior change, but will help to make sure that behavior does not change unintentionally in future.